scripts/Install-VSCode.ps1

```diff
@@ -56,6 +56,10 @@
     A validated string defining which build edition or "stream" to download - stable or
     insiders edition. If the parameter is not used, then stable is downloaded as default.
 
+.PARAMETER User
```

Comment:
The main function of this flag is to install VSCode-Insiders (user edition) right? Instead of non-Insiders edition. If that's the case, I would vote for the flag being called something like `Insiders` for now, and then if we want to make it more complex (e.g. choose between User Profile and non-User Profile) later, we could add another flag.

Reply:
@rjmholt I agree with you. In that case would it make sense for now then to move from the switch into the `$BuildEdition` parameter as a validation value?

Something like: `[ValidateSet("stable","insider-system","insider-userprofile")]` ??

Reply:
Yeah that's a really good idea!